iree/vm/BUILD

```diff
@@ -61,7 +61,9 @@ cc_library(
         "//iree/base:api",
         "//iree/base:flatbuffer_util",
         "//iree/base:target_platform",
+        "//iree/schemas:bytecode_module_def_c_fbs",
         "//iree/schemas:bytecode_module_def_cc_fbs",
+        "@com_github_dvidelabs_flatcc//:runtime",
```

Comment:
To help with integration, usage could be deferred to a second PR, or left commented out at first